### PR TITLE
changing http to https

### PR DESCRIPTION
**Pull request by Lux-Sales**
Closes # Doesn't apply

fix

### Technical

Just change the protocol that acces the endpoint, from http to https, to implements the security

### CODE REVIEW

**Comment by hornc** [comment]
Thanks for the PR @Lux-Sales , but these changes are incorrect; they change and break the namespaces.

**TL;DR**: `xmlns` declarations are XML namespace names *NOT* URLs. `http` -> `https` is good, but only applies to URLs.


The strings changed are XML namespace names (`xmlns:`) and they are specific strings that cannot be changed without changing the meaning. They look like URLs, but they do not have to function as locators.

The [spec](https://www.w3.org/TR/2009/REC-xml-names-20091208) says they have to be URIs (which includes URLs, as well as URNs) -- I personally don't understand why it doesn't say they have to be *URN*s.

From the section [Declaring Namespaces](https://www.w3.org/TR/2009/REC-xml-names-20091208/#ns-decl):

>The namespace name, to serve its intended purpose, SHOULD have the characteristics of uniqueness and persistence. It is not a goal that it be directly usable for retrieval of a schema (if any exists). Uniform Resource Names [[RFC2141]](https://www.w3.org/TR/2009/REC-xml-names-20091208/#URNs) is an example of a syntax that is designed with these goals in mind. However, it should be noted that ordinary URLs can be managed in such a way as to achieve these same goals.

I can't think of an example where a URL "managed in such a way as to achieve these same goals" ***isn't*** a URN, but the spec ***is*** clear on how the xmlns string comparison works in [Comparing URI References](https://www.w3.org/TR/2009/REC-xml-names-20091208/#NSNameComparison):

> [Definition: The two URIs are treated as strings, and they are identical if and only if the strings are identical, that is, if they are the same sequence of characters. ]

I think this is the ultimate reason why changing `http` to `https` is incorrect here.

This `http` / `https` thing is confusing when it comes to XML namespaces. I've had to look this up before and found it's hard to get a clear statement on what should be done any why. The latest XML namespace spec was written in 2009, before https was everywhere.

This link is pretty good for it's accepted answer: https://stackoverflow.com/questions/30707609/xml-namespace-uri-with-https  but still could be clearer. I used the spec links from there to compose this (partly for my own future reference too).

Hope that helps, I've wanted to get this straight in my head too, so thanks for prompting me to look into it.

**Comment by mekarpeles** [comment]
Closing in agreement w/ @hornc's analysis -- thank you for submitting this PR for review and surfacing this explanation!
